bentoml/_internal/frameworks/pytorch.py

```diff
@@ -170,10 +184,12 @@ def forward(self, inputs):
     )
     weight_file = _model.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
     if isinstance(model, torch.jit.ScriptModule):  # type: ignore[reportPrivateUsage]
+        _model.info.options["_model_format"] = "torchscript:v1"
         torch.jit.save(model, weight_file)  # type: ignore[reportUnknownMemberType]
     else:
+        _model.info.options["_model_format"] = "torch.save:v1"
```

Comment:
Hmm on a second thought, this is only our internal format right? I don't think with the current implementation allows us to modify `context` at bento model creation time